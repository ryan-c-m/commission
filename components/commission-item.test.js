import 'react-native';
import React from 'react';
import CommissionItem from './commission-item';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <CommissionItem />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});

