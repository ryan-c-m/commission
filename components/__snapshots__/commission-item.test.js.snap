// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "maxHeight": 100,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#000000",
        "fontFamily": "sans-serif",
        "fontSize": 20,
      }
    }
  />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderBottomWidth": 1.5,
        "borderColor": "rgb(211,211,211)",
        "borderRadius": 5,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "flex-end",
        "maxWidth": 120,
        "padding": 5,
      }
    }
    testID={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#bbbbbb",
          "fontFamily": "sans-serif",
          "fontSize": 20,
        }
      }
    >
      $
    </Text>
    <TextInput
      allowFontScaling={true}
      keyboardType="phone-pad"
      multiline={false}
      onChangeText={[Function]}
      placeholder="0"
      placeholderTextColor="#bbbbbb"
      selectTextOnFocus={true}
      style={
        Object {
          "color": "#000000",
          "fontSize": 20,
          "margin": 5,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
</View>
`;
